Wikipedia/UI-V5/WMFEditingCollectionViewLayout.m

```diff
@@ -371,6 +371,11 @@ - (IBAction)panWithGesture:(UIPanGestureRecognizer*)pan {
                 CGPoint currentCenter = self.movingCellCenter;
                 currentCenter.x               += translation.x;
                 self.movingSnapshotView.center = currentCenter;
+                CGFloat alpha = CGRectGetMaxX(self.movingSnapshotView.frame) / self.collectionView.bounds.size.width;
+                if(alpha > 1){
+                    alpha = 1 - (alpha - 1);
```

Comment:
can you add some comments explaining what is going on here? it looks like you're "capping" the alpha at 1, but it's not clear why you're subtracting the "remainder" above 1 from 1.